pages/partials/navTabs/institutionAdmin.ejs

```diff
@@ -0,0 +1,10 @@
+<%- include('navTab', {
```

Comment:
Not that it really matters, but if all the institution admin stuff is going under `ee/` (which I think it should) then should this file go in `ee/partials` or similar? It's also fine if it's more convenient to keep it where it is.

Reply:
I'll leave these where they are for now, I don't think there's a ton of value in moving them. We can easily move in the future if we develop different conventions for EE code.

Reply:
Makes sense!